src/data/fintype/basic.lean

```diff
@@ -534,6 +534,13 @@ arbitrary `fintype` instances, use either `fintype.card_le_one_iff_subsingleton`
   fintype.card α = 1 :=
 subsingleton.elim (of_subsingleton $ default α) h ▸ card_of_subsingleton _
 
+@[simp] lemma card_subtype_eq (y : α) [fintype {x // x = y}] :
```

Comment:
I assume the `fintype` argument here is more to prevent confusion over different defeq instances rather than because the instance can't be inferred, right?